src/core/a-animation.js

```diff
@@ -499,6 +500,31 @@ function boolToNum (bool) {
 }
 
 /**
+ * A getComputedAttribute that supports dot notation to look up a component property.
+ *
+ * @param {Element} el - To look up attribute on.
+ * @param {string} attr - dot notation or singular 'color' or 'material.color'
+ * @returns Resulting value of component property.
+ */
+function getComputedAttributeFor (el, attribute) {
+  var attributeSplit = attribute.split('.');
+  var componentName = attributeSplit[0];
+  var componentPropName = attributeSplit[1];
+  // If the attribute is singular call normal getComputedAttribute function
+  if (attributeSplit.length === 1) {
+    return el.getComputedAttribute(componentName);
+  }
+  // If the component exists as an attribute return the property on it
+  if (el.getComputedAttribute(componentName)) {
```

Comment:
@dmarcos I may be misunderstanding, but this doesn't work in the case of an attribute needing to be split.  We need 3 cases here one for a default, one for a attribute split and then fallback to the native if it doesn't work.